kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -173,6 +175,9 @@
         stopTracking: stopTrackingProgress,
         genContentLinkKeepCurrentBackLink,
         windowIsSmall,
+        isUserLoggedIn,
+        currentUserId,
+        session,
```

Comment:
Can just directly get the `full_name` attribute here

https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/composables/useUser.js#L59